exports/defect_dojo.py

```diff
@@ -19,13 +19,30 @@ def __init__(self, base_url, username=None, password=None, token=None):
             raise ValueError("Defect Dojo invalid configuration: URL is not correct")
 
         self.base_url = base_url
-        self.username = username
-        self.password = password
+
+        self.username = None
+        self.password = None
+        if login:
+            try:
+                self.username = login["username"]
+                self.password = login["password"]
+            except KeyError:
+                logging.error(
+                    "RapiDAST BUG: DefectDojo was created with invalid login information..."
```

Comment:
'RapiDAST BUG' sounds like "RapiDAST has a bug", which is not the case here. 'BUG' should be removed

Reply:
If this happens, it's the caller's fault. This should not be an issue, but I only wanted to highlight that if it ever happens, it's because of an issue in the code and not in the configuration.

For the details: in our current pylint, we accept up to 5 arguments max. So I had to combine username/password into a single argument to remain within the limits. 
Alternatively, I could tell pylint to ignore the argument limit on this function (or we increase it globally), to avoid this combining several arguments into one and get back to the old code for username/password